src/components/victory-candlestick/helper-methods.js

```diff
@@ -19,10 +19,12 @@ export default {
       const y2 = scale.y(datum._low);
       const candleHeight = Math.abs(scale.y(datum._open) - scale.y(datum._close));
       const y = scale.y(Math.max(datum._open, datum._close));
+      const highWick = y;
+      const lowWick = scale.y(Math.min(datum._open, datum._close));
       const dataStyle = this.getDataStyles(datum, style.data, props);
       const dataProps = {
-        x, y, y1, y2, candleHeight, scale, data, datum, groupComponent,
-        index, style: dataStyle, padding, width, polar, origin
+        x, y, y1, y2, candleHeight, scale, data, datum, groupComponent, highWick, lowWick,
```

Comment:
Looking at this now, and thinking about how you'll actually be using these props, I think it would be nice to refactor the props that get supplied to `Candle`. Rather than `x`, `y`, `y1`, `y2`, I think it makes sense to call these variables `x` `high`, `low`, `open`, `close`. I think they will be easier to work with that way, and make more sense to users who want to write custom candle components.